content/docs/hoon/guides/sail.md

```diff
@@ -0,0 +1,495 @@
+```
+
+### Marks
+
+The mark for a raw `$manx` is a `%hymn`. This mark is included in the `%base`
+desk. The mark for a rendered HTML cord is `%html`. The `%hymn` mark includes
+conversion methods to both `%html` and `%mime`. The `%html` mark includes
+conversion methods to `%mime` and `%hymn`.
+
+If you want to produce an HTML document through Eyre's scry interface, for
+example, you can either produce a `$manx` with a `%hymn` mark and let Eyre
+convert it to `%html`, or else you can call `++en-xml:html` and then `++crip` on
+the product of your Sail expressions and produce a cord with an `%html` mark
+directly.
+
+## Sail Runes
+
+In addition to the syntax so far described, there are also a few Sail-specific
+runes:
+
+### `;+` Miclus
+
+The [miclus rune](/docs/hoon/reference/rune/mic#-miclus) makes a `$marl` from a
+complex hoon expression that produces a single `$manx`. Its main use is nesting
+tall-form hoon logic in another Sail element. For example:
+
+```hoon
+;p
+  ;b: {(a-co:co number)}
+  ; is an
+  ;+  ?:  =(0 (mod number 2))
+        ;b: even
+      ;b: odd
+  ; number.
+==
+```
+
+Produces one of these depending on the value of `number`:
+
+```
+<p><b>2 </b>is an <b>even </b>number.</p>
+```
+
+```
+<p><b>12345 </b>is an <b>odd </b>number.</p>
+```
+
+### `;/` Micfas
+
+The [micfas rune](/docs/hoon/reference/rune/mic#-micfas) turns an ordinary tape
+into a `$manx`. For example:
+
+```
+> %-  en-xml:html  ;/  "foobar"
+"foobar"
+```
+
+In order to nest it inside another Sail element, it must be preceeded with a
+`;+` rune or similar, it cannot be used directly. For example:
+
+```hoon
+;p
+  ;+  ;/  "foobar"
+==
+```
+
+The argument for Micfas needn't directly be a tape, it may also be a hoon
+expression that produces a tape.
+
+### `;*` Mictar
+
+The [mictar rune](/docs/hoon/reference/rune/mic#-mictar) makes a `$marl` (a list
+of XML nodes) from a complex hoon expression. This rune lets you add many
+elements inside another Sail element. For example:
+
+```hoon
+=/  nums=(list @ud)  (gulf 1 9)
+;p
+  ;*  %+  turn  nums
+      |=  n=@ud
+      ?:  =(0 (mod n 2))
+        ;sup: {(a-co:co n)}
+      ;sub: {(a-co:co n)}
+==
+```
+
+Produces:
+
+```
+<p>
+  <sub>1</sub><sup>2</sup><sub>3</sub>
+  <sup>4</sup><sub>5</sub><sup>6</sup>
+  <sub>7</sub><sup>8</sup><sub>9</sub>
+</p>
+```
+
+### `;=` Mictis
+
+The [mictis rune](/docs/hoon/reference/rune/mic#-mictis) makes a `$marl` (a list
+of XML nodes) from a series of `$manx`es. This is mostly useful if you want to
+make the list outside of an element and then be able to insert it afterwards.
+For example:
+
+```hoon
+=/  paras=marl
+  ;=  ;p: This is the first node.
+      ;p: This is the second.
+      ;p: Here is the last one.
+  ==
+;main
+  ;*  paras
+==
+```
+
+Produces:
+
+```
+<main>
+  <p>This is the first node.</p>
+  <p>This is the second.</p>
+  <p>Here is the last one.</p>
+</main>
+```
```

Comment:
Let's have a section of "Good Examples" with links to projects like %pals and %gora.